src/main/java/fi/aalto/cs/apluscourses/model/CourseVersion.java

```diff
@@ -0,0 +1,51 @@
+package fi.aalto.cs.apluscourses.model;
+
+import org.jetbrains.annotations.NotNull;
+import org.json.JSONObject;
+
+public class CourseVersion {
+
+  public enum Status {
+    VALID,
+    UPDATE_OPTIONAL,
+    UPDATE_REQUIRED
+  }
+
+  // the major and minor version of the current plugin build
+  private static final int MAJOR_VERSION = 1;
+  private static final int MINOR_VERSION = 0;
```

Comment:
Remark: So, if I understood correctly, whenever the course config file-format is extended/changed, these constants should be increased, too. Not sure, if this is the most obvious place to maintain these version numbers.